ent/encryption/encryption.cc

```diff
@@ -823,6 +823,21 @@ class encryption_file_io_extension : public sstables::file_io_extension {
         });
         co_return sink;
     }
+
+    future<data_source> wrap_source(const sstables::sstable& sst, sstables::component_type type, data_source source) override {
+        switch (type) {
+        case sstables::component_type::Scylla:
+        case sstables::component_type::TemporaryTOC:
+        case sstables::component_type::TOC:
+            co_return source;
+        default:
+            break;
+        }
+        co_await wrap_writeonly(sst, type, [&source](shared_ptr<symmetric_key> k) {
```

Comment:
What does `writeonly` mean? The method name is generic, it doesn't seem to be used just for writes.

Reply:
Right, no idea. @elcallio any idea what makes this wrapper "write only"